src/main.rs

```diff
@@ -174,8 +175,27 @@ fn main() {
                         match rx.recv() {
                             Ok(val) => {
                                 trace!("file changed {:?}", val);
-                                info!("Rebuilding cobalt site...");
-                                build(&config);
+
+                                if let Some(path) = val.path {
+                                    if path.is_absolute() {
+                                        // get where process was run from
+                                        let cwd = std::env::current_dir().unwrap_or(PathBuf::new());
+                                        // strip absolute path
+                                        let rel_path = path.strip_prefix(&cwd).unwrap_or(&cwd);
+
+                                        // check if path starts with the build folder.
+                                        if !rel_path.starts_with(&config.dest) {
```

Comment:
also, the comment should be updated, you want to check if we're _not_ in the dest folder